### DISCUSSION TITLE
Add support for ISR.

### DISCUSSION THREAD

**Discussion by cyborgdennett** [bxparks/UnixHostDuino#79] [Ideas]
Hi I have a feature request. I don't know how hard it would be but is it possible to add ISR from <avr/interrupts.h> and <avr/io.h>? Would be nice to test in this environment.

**Comment by bxparks** [comment]
Please use the Discussions for feature requests, as explained here:
https://github.com/bxparks/EpoxyDuino#FeedbackAndSupport
(I add this note because this is your second feature request made in Issues.)

**Comment by bxparks** [comment]
@cyborgdennett: Can you expand on what ISR functionality you want to emulate in EpoxyDuino? How will the ISR be triggered in a Unix environment?
